OpenCL_API.txt

```diff
@@ -16,7 +16,9 @@ Khronos{R} OpenCL Working Group
 :source-highlighter: rouge
 :source-language: opencl
 :rouge-style: opencl.spec
-:title-logo-image: image:images/OpenCL.png[Logo,pdfwidth=4in,align=right]
+:docinfo: shared-header
+:docinfodir: config
+:title-logo-image: image:images/OpenCL.png[top="25%"]
```

Comment:
```suggestion
:title-logo-image: image:images/OpenCL.png[top="25%",width="55%"]
```

Having the logo being the entire width of the page in the PDF seems quite large.  55% width is pretty arbitrary, it's close to what is in the current PDF render.  To me this looks a little better.